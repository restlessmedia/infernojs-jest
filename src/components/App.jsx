import { Component } from 'inferno';
import Animated from './Animated';

class App extends Component {
  render() {
    return (
      <div>
        App
        <Animated />
      </div>
    )
  }
}

export default App;