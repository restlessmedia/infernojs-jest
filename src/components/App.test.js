import App from './App';
import { renderToSnapshot } from 'inferno-test-utils';

describe('App', () => {

  it('should render', () => {
    const props = {};
    const renderedTree = renderToSnapshot(<App {...props} />);

    expect(renderedTree).toMatchSnapshot();
  });
});
