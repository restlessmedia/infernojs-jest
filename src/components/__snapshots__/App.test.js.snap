// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render 1`] = `
<div>
  App
  <div
    className="animated bounceIn "
  />
</div>
`;
